simulator/pkg/externalscheduler/external_scheduler.go

```diff
@@ -26,83 +26,87 @@ import (
 	simulatorconfig "sigs.k8s.io/kube-scheduler-simulator/simulator/config"
```

Comment:
We don't need to think about the external scheduler package anymore. If people want to use a new feature by this feature, they can just move to debuggable scheduler package.